lib/miq_automation_engine/engine/miq_ae_service/miq_ae_service_rbac.rb

```diff
@@ -0,0 +1,34 @@
+module MiqAeMethodService
+  module MiqAeServiceRbac
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def find_ar_object_by_id(id)
+        Rbac.search(:class => model, :targets => [id],
+                    :results_format => :objects).try(:first).try(:first)
+      end
+
+      def all
+        wrap_results(Rbac.search(:class => model, :results_format => :objects).try(:first))
+      end
+
+      def count
+        Rbac.search(:class => model).try(:first).count
+      end
+
+      def first
+        all.first
+      end
+
+      def filter_objects(objs)
+        return objs if objs.nil?
```

Comment:
in rbac, working towards just using scopes.

The support of arrays and stuff begs to ask why we're doing it that way.
Also, do we need to call all this rbac logic for a single object? would be nice if this logic were executed in rub